src/complete/buildin.rs

```diff
@@ -1,10 +1,37 @@
 /// buildin Commands and vars
 use anyhow::Result;
 use once_cell::sync::Lazy;
+use std::env;
 use std::process::Command;
 use std::{collections::HashMap, iter::zip};
 use tower_lsp::lsp_types::{CompletionItem, CompletionItemKind};
 
+pub static CMAKE_MINIMAL: Lazy<bool> = Lazy::new(|| {
+    let key = "NEOCMAKELSP_MINIMAL";
```

Comment:
you should use https://github.com/Decodetalkers/neocmakelsp/blob/master/src/languageserver/config.rs lsp config to control it, not environment, I think..